lib/serial_terminal.pm

```diff
@@ -354,6 +374,7 @@ sub select_serial_terminal {
     }
 
     die "No support for backend '$backend', add it" if (!defined $console) || ($console eq '');
+    $testapi::distri->{serial_term_prompt} = $prompt;
```

Comment:
Together with this call. Or should be prompt be handled in `select_console()` (optionally, probably part of another effort)?